packages/client/hmi-client/src/services/project.ts

```diff
@@ -57,9 +57,15 @@ async function getAll(): Promise<Project[] | null> {
  * Get project assets for a given project per id
  * @return ProjectAssets|null - the appropriate project, or null if none returned by API
  */
-async function getAssets(projectId: string): Promise<ProjectAssets | null> {
+async function getAssets(projectId: string, types?: string[]): Promise<ProjectAssets | null> {
 	try {
-		const response = await API.get(`/projects/${projectId}/assets`);
+		let url = `/projects/${projectId}/assets`;
+		if (types) {
+			types.forEach((type, indx) => {
+				url += `${indx === 0 ? '?' : '&'}types=${type}`;
+			});
```

Comment:
It might be more readable with an [`Array.join()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join)
```js
url += `?type=${types.join('&type='}`;
```

Reply:
It has to be ?types=A&types=B&types=C unfortunately. I will add comments to this to help its readability